### DISCUSSION TITLE
How can I plot the result of distributions from two models with the same sample on the same figure?

### DISCUSSION THREAD

**Discussion by DianaY-J** [mwshinn/PyDDM#103] [Ask for help] [answered]
Hi folks, 

I'm using two gddm models with slightly different parameters to fit my data, and i want to compare their performance by visualising the distribution on the same figure, but I don't know how to do it. I want to look into the function plot_fit_diagnostics, but it's empty for me (https://pyddm.readthedocs.io/en/stable/apidoc/plotting.html) like the pic attached below. 
<img width="1248" alt="Screenshot 2024-06-24 at 15 18 22" src="https://github.com/mwshinn/PyDDM/assets/121393800/596a1bb2-e94f-43c1-8355-41efc4a48b09">

How can I put distributions of two models on the same plot? I'm expecting something like this (I manually drew the green line but let me know if it's unclear to you)
![mylineargddm-fit](https://github.com/mwshinn/PyDDM/assets/121393800/c828321e-88f1-4f3f-99b3-f50a9a1305e5)

**Comment by mwshinn** [comment, marked as answer]
Thanks for pointing out that the documentation here was empty, this was a problem with the documentation build server should be fixed now!

https://pyddm.readthedocs.io/en/latest/apidoc/plotting.html

You'll have to write the function you want since it does not exist, but a good place to start is plot_fit_diagnostics.

**Reply by DianaY-J** [reply]
Omg thank you so much!

**Comment by delphinepape** [comment]
Were you able to figure out how to write the function? As of now nothing has worked for me
